waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/security/AccessControlHandlerFactory.java

```diff
@@ -31,7 +32,7 @@
   private final FederationService federationService;
 
   @Autowired
-  public AccessControlHandlerFactory(FederationService federationService) {
+  public AccessControlHandlerFactory(@Qualifier("notifyingFederationService") FederationService federationService) {
```

Comment:
There are 2 implementations of `FederationService` so I've added the Qualifiers to make sure we pick the correct one where we need it.